### PR TITLE
Vg 43 finish contact page

### PR DESCRIPTION
**Pull request by Danback**
* Added styling to contact us page
* Added functionality to contact us page
* "Customers" can now send emails to "us"

### CODE REVIEW

File: src/views/ContactView.vue [hunk 1]
```diff
@@ -1,14 +1,73 @@
...
+      </p>
     </div>
-  </main>
-</template>
\ No newline at end of file
+
+    <div class="bg-white shadow-lg rounded-lg p-6 max-w-2xl mx-auto">
+      <form @submit.prevent="submitForm">
+        <div class="mb-4">
+          <label for="name" class="block text-sm font-medium text-gray-700">Your Name</label>
```
Comment (marcusju23) [inline src/views/ContactView.vue]: Remove redundant "Your" on label.

File: src/views/ContactView.vue [hunk 2]
```diff
@@ -1,14 +1,73 @@
...
+              v-model="name"
+              id="name"
+              class="mt-1 block w-full p-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring focus:ring-green-500"
+              placeholder="John Doe"
+              required
+          />
+        </div>
+
+        <div class="mb-4">
+          <label for="email" class="block text-sm font-medium text-gray-700">Your Email</label>
```
Comment (marcusju23) [inline src/views/ContactView.vue]: Remove redundant "Your" on label.

File: src/views/ContactView.vue [hunk 3]
```diff
@@ -1,14 +1,73 @@
...
+              v-model="email"
+              id="email"
+              class="mt-1 block w-full p-2 border border-gray-300 rounded-md shadow-sm focus:outline-none focus:ring focus:ring-green-500"
+              placeholder="you@example.com"
+              required
+          />
+        </div>
+
+        <div class="mb-6">
+          <label for="message" class="block text-sm font-medium text-gray-700">Your Message</label>
```
Comment (marcusju23) [inline src/views/ContactView.vue]: Remove redundant "Your" on label.

**Review by marcusju23 — CHANGES_REQUESTED** [review]
Well done! Just minor changes and it's good to go 👍

**Review by marcusju23 — APPROVED** [review]

**Review by Rishad11 — APPROVED** [review]
Nicely done, Daniel!

**Pull request merged**